cvs diff: Diffing scp2
cvs diff: Diffing scp2/inc
cvs diff: Diffing scp2/macros
cvs diff: Diffing scp2/prj
cvs diff: Diffing scp2/source
cvs diff: Diffing scp2/source/activex
cvs diff: Diffing scp2/source/base
cvs diff: Diffing scp2/source/calc
cvs diff: Diffing scp2/source/canvas
cvs diff: Diffing scp2/source/crashrep
cvs diff: Diffing scp2/source/draw
cvs diff: Diffing scp2/source/gnome
cvs diff: Diffing scp2/source/graphicfilter
cvs diff: Diffing scp2/source/impress
cvs diff: Diffing scp2/source/javafilter
cvs diff: Diffing scp2/source/kde
cvs diff: Diffing scp2/source/layout
Index: scp2/source/layout/layout.scp
===================================================================
RCS file: /cvs/installation/scp2/source/layout/layout.scp,v
retrieving revision 1.2
retrieving revision 1.2.20.3
diff -p -u -u -p -b -w -B -r1.2 -r1.2.20.3
--- scp2/source/layout/layout.scp	6 Mar 2008 12:47:00 -0000	1.2
+++ scp2/source/layout/layout.scp	31 Mar 2008 14:53:28 -0000	1.2.20.3
@@ -35,83 +35,21 @@
 
 #include "macros.inc"
 
-File gid_File_Lib_Layout
-    TXT_FILE_BODY;
-    Styles = (PACKED,UNO_COMPONENT);
-    RegistryID = gid_Starregistry_Services_Rdb;
-    Dir = gid_Dir_Program;
-  #ifdef UNX
-    Name = STRING(CONCAT4(libtklayout,OFFICEUPD,DLLPOSTFIX,UNXSUFFIX));
-  #else
-    Name = STRING(CONCAT4(tklayout,OFFICEUPD,DLLPOSTFIX,.dll));
-  #endif
-End
-
 Directory gid_Dir_Share_Layout
     ParentID = gid_Dir_Share;
     DosName = "layout";
 End
 
-File gid_File_Xml_Wordcount
-   TXT_FILE_BODY;
+File gid_File_Zip_Sw_Layout
    Dir = gid_Dir_Share_Layout;
-   Name = "wordcount.xml";
-   Styles = (PACKED);
+   USER_FILE_BODY;
+   Styles = (ARCHIVE, PATCH);
+   Name = "sw-layout.zip";
 End
 
-File gid_File_Xml_Zoom
-   TXT_FILE_BODY;
+File gid_File_Zip_Svx_Layout
    Dir = gid_Dir_Share_Layout;
-   Name = "zoom.xml";
-   Styles = (PACKED);
-End
-
-Directory gid_Dir_Share_Layout_De
-    ParentID = gid_Dir_Share_Layout;
-    DosName = "de";
-End
-
-File gid_File_Xml_Wordcount_De
-   TXT_FILE_BODY;
-   Dir = gid_Dir_Share_Layout_De;
-   Name = "de/wordcount.xml";
-   Styles = (PACKED);
-End
-
-File gid_File_Xml_Zoom_De
-   TXT_FILE_BODY;
-   Dir = gid_Dir_Share_Layout_De;
-   Name = "de/zoom.xml";
-   Styles = (PACKED);
+   USER_FILE_BODY;
+   Styles = (ARCHIVE, PATCH);
+   Name = "svx-layout.zip";
 End
-
-Directory gid_Dir_Share_Layout_Nl
-    ParentID = gid_Dir_Share_Layout;
-    DosName = "nl";
-End
-
-File gid_File_Xml_Wordcount_Nl
-   TXT_FILE_BODY;
-   Dir = gid_Dir_Share_Layout_Nl;
-   Name = "nl/wordcount.xml";
-   Styles = (PACKED);
-End
-
-File gid_File_Xml_Zoom_Nl
-   TXT_FILE_BODY;
-   Dir = gid_Dir_Share_Layout_Nl;
-   Name = "nl/zoom.xml";
-   Styles = (PACKED);
-End
-
-//  Do we need this?
-//  Must MODULE_OPTIONAL_LAYOUT be registered somewhere?
-//
-// Module gid_Module_Layout
-//    MOD_NAME_DESC (MODULE_OPTIONAL_LAYOUT);
-//    ParentID = gid_Module_Optional;
-//    Files = (gid_File_Lib_Layout,gid_File_Xml_Wordcount,gid_File_Xml_Zoom);
-//    Minimal = NO;
-//    Default = YES;
-//    Styles = ();
-// End
cvs diff: Diffing scp2/source/lingu
cvs diff: Diffing scp2/source/math
cvs diff: Diffing scp2/source/onlineupdate
cvs diff: Diffing scp2/source/ooo
cvs diff: Diffing scp2/source/ooo_wo_ure
cvs diff: Diffing scp2/source/python
cvs diff: Diffing scp2/source/quickstart
cvs diff: Diffing scp2/source/sdkoo
cvs diff: Diffing scp2/source/templates
cvs diff: Diffing scp2/source/testtool
cvs diff: Diffing scp2/source/ure
cvs diff: Diffing scp2/source/winexplorerext
cvs diff: Diffing scp2/source/writer
cvs diff: Diffing scp2/source/xsltfilter
cvs diff: Diffing scp2/util
